changelog.txt

```diff
@@ -1,5 +1,5 @@
 == Changelog ==
-
+* Fix - Exporting Attendees with HTML entities [ETP-730]
```

Comment:
```suggestion

* Fix - Prevent Attendees with HTML entities from importing broken [ETP-730]

```

Reply:
Please also add a stop before the ticket ID.